app/assets/stylesheets/application.scss

```diff
@@ -338,7 +338,7 @@ ul.no_bullet_list li {
 }
 
 .yes_title {
-  cursor: help;
+  cursor: default;
```

Comment:
You can just remove this line completely since `cursor: default` is the default behaviour.